src/containers/Layout/MainLayout.tsx

```diff
@@ -53,15 +54,29 @@ const MainLayoutWrapper = styled.div`
 `;
 
 const UstWarning = styled.div`
-  background-color: rgba(233, 61, 68, 1);
+  background-color: rgba(255, 231, 206, 1);
   width: 100%;
   padding: 4px 16px;
   justify-content: center;
   align-items: center;
   display: inline-flex;
+  height: 56px;
 
   p {
-    color: rgba(255, 255, 255, 1);
+    color: rgba(0, 0, 0, 1);
```

Comment:
so and this one

Reply:
Since this is a deprecated component not using our new them I hardcoded them. This whole component will be removed shortly.